### PR TITLE
kintsugi: Fix passing options from install-driver to driver.

### PR DESCRIPTION
**Pull request by byohay**

### CODE REVIEW

**Review by ashdnazg — APPROVED** [review]
LGTM

File: lib/kintsugi/cli.rb [hunk 1]
```diff
@@ -106,8 +104,9 @@ def create_install_driver_subcommand
     end
 
     def install_kintsugi_driver_globally(options)
+      joined_options = options.count > 1 ? " #{options.join(" ")}" : ""
```
Comment (ashdnazg) [inline lib/kintsugi/cli.rb:107]: shouldn't this just be `options.join(" ")`
at the moment it ignores options if it has just one element

Comment (byohay) [reply lib/kintsugi/cli.rb:107]: oops, it's supposed to be `options.count > 0`. The problem is that when is there are no options a trailing space is added which writes the driver command incorrectly: 
`driver = "kintsugi driver %O %A %B %P "`

**Review by ashdnazg — APPROVED** [review]
LGTM

**Pull request merged**